css-color/Overview.bs

```diff
@@ -1714,7 +1717,7 @@ Profiled, Device-dependent Colors</h2>
 	or an RGB colorspace (such as ProPhoto <!-- ref -->, <!-- would be good to mention AdobeRGB 1998 here too, need to check trademark and copyright issues first --> widely used by photographers), or any other color or monochrome output device which has been characterized.
 	In addition, for convenience,
 	CSS provides two predefined RGB color spaces:
-	DCI P3 [[!DCI-P3]],
+	p3d65 [[!DCI-P3]],
```

Comment:
I was told that these are the values that are currently used by Apple in the p3 profile.
I can send that profile to the group if wanted so you can double-check